src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -0,0 +1,548 @@
+# Changelog
+
+* Update `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.8.1`.
+  * Update `OpenTelemetry.Api` to `1.8.1`.
```

Comment:
do we need to mention `OpenTelemetry.Api` separately. We only target `OpenTelemetry.Api.ProviderBuilderExtensions`

Reply:
It is transitive dependency. Entry is copied from SqlClient, before I copied it from another package referencing exactly this package.

We can fix it all at once in separate PR if you want.